versions/1.0.0.md

```diff
@@ -250,8 +257,8 @@ Field Name | Type | Description
 <a name="workflowId"></a>workflowId | `string` | **REQUIRED**. Unique string to represent the workflow. The id MUST be unique amongst all workflows describe in the Workflows document. The `workflowId` value is **case-sensitive**. Tools and libraries MAY use the `workflowId` to uniquely identify a workflow, therefore, it is RECOMMENDED to follow common programming naming conventions.
 <a name="workflowSummary"></a>summary | `string` | A summary of the purpose or objective of the workflow.
 <a name="workflowDescription"></a>description | `string` | A description of the workflow. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
-<a name="workflowInputs"></a>inputs | `JSON Schema` | A JSON Schema object representing the input parameters used by this workflow
-<a name="workflowSteps"></a>steps | [[Step Object](#step-object)] | **REQUIRED**. An ordered list of steps where each step represents a call to an API operation or to another workflow
+<a name="workflowInputs"></a>inputs | `JSON Schema` | A JSON Schema 2020-12 object representing the input parameters used by this workflow.
+<a name="workflowSteps"></a>steps | [[Step Object](#step-object) | [Reference Object](#reference-object)] or `string` | **REQUIRED**. An ordered list of steps where each step represents a call to an API operation or to another workflow. If a Reference Object is provided, it MUST refer to a Step.

```

Comment:
Some formatting issues here. I would state as follows as below if proceeding but I'm not sure if we decided to hold back from referencing _steps_ for now?
```suggestion
<a name="workflowSteps"></a>steps | [[Step Object](#step-object) \| [Reference Object](#reference-object)] | **REQUIRED**. An ordered list of steps where each step represents a call to an API operation or to another workflow. The list can use the [Reference Object](#referenceObject) to link to steps that are defined at the [Workflows Object's components/parameters](#component-object). If a Reference Object is provided, it MUST refer to a Step.

```

Reply:
Yeah I thought we were knocking out Steps in favor of just using Workflows as the unit of reuse. My pet store example shows how to do that.